### PR TITLE
GH-672 review two

### PR DESCRIPTION
**Pull request by bertllll**

### CODE REVIEW

File: masq_lib/src/blockchains/blockchain_records.rs [hunk 1]
```diff
@@ -16,6 +16,10 @@ pub const CHAINS: [BlockchainRecord; 5] = [
         self_id: Chain::PolyMainnet,
         num_chain_id: 137,
         chain_family: ChainFamily::Polygon,
+        transaction_tech_specifications: TransactionTechSpecs {
```
Comment (bertllll) [inline masq_lib/src/blockchains/blockchain_records.rs]: Maybe just `transaction_specifications` ?

Comment (dnwiebe) [reply masq_lib/src/blockchains/blockchain_records.rs]: Or `transaction_fee_specifications`?

Comment (dnwiebe) [inline masq_lib/src/blockchains/blockchain_records.rs]: Should this data be part of the `BlockchainInterface`, for when it becomes a `Web3BlockchainInterface` implementation of the new `BlockchainInterface` trait?

File: masq_lib/src/blockchains/blockchain_records.rs [hunk 2]
```diff
@@ -48,22 +64,43 @@ pub const CHAINS: [BlockchainRecord; 5] = [
...
 
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct TransactionTechSpecs {
+    //we picked this value out empirically unless commented differently
+    pub base_of_required_transaction_fee_units: u64,
+    pub maximum_of_required_transaction_fee_units: u64,
+}
+
+#[derive(Debug, PartialEq, Eq, Copy, Clone, Hash)]
+pub enum ChainFamily {
```
Comment (dnwiebe) [inline masq_lib/src/blockchains/blockchain_records.rs]: This enum may be outdated.

File: node/src/accountant/scanners/mod.rs [hunk 3]
```diff
@@ -1112,7 +1132,8 @@ mod tests {
         payable_scanner
             .payment_adjuster
             .as_any()
-            .downcast_ref::<PaymentAdjusterReal>();
+            .downcast_ref::<PaymentAdjusterReal>()
```
Comment (dnwiebe) [inline node/src/accountant/scanners/mod.rs]: This assertion may not be necessary

File: node/src/blockchain/blockchain_interface.rs [hunk 4]
```diff
@@ -195,8 +194,8 @@ impl BlockchainInterface for BlockchainInterfaceClandestine {
         Err(BlockchainError::QueryFailed(msg))
     }
 
-    fn estimated_gas_limit_per_payable(&self) -> u64 {
-        blockchain_interface_common::base_gas_limit(self.chain) + TRANSACTION_DATA_GAS_MARGIN_MAX
+    fn chain(&self) -> Chain {
+        todo!()
```
Comment (dnwiebe) [inline node/src/blockchain/blockchain_interface.rs:200]: Forgotten `todo!`?

File: node/src/sub_lib/accountant.rs [hunk 5]
```diff
@@ -95,7 +95,7 @@ pub struct AccountantSubs {
     pub report_routing_service_provided: Recipient<ReportRoutingServiceProvidedMessage>,
     pub report_exit_service_provided: Recipient<ReportExitServiceProvidedMessage>,
     pub report_services_consumed: Recipient<ReportServicesConsumedMessage>,
-    pub report_consuming_wallet_balances_and_qualified_payables: Recipient<PayablePaymentSetup>,
+    pub report_payable_payment_setup: Recipient<PayablePaymentsSetup>,
```
Comment (dnwiebe) [inline node/src/sub_lib/accountant.rs]: Should be `report_payable_payments_setup`: plural

File: node/src/test_utils/mod.rs [hunk 6]
```diff
@@ -886,7 +886,7 @@ pub mod unshared_test_utils {
                 self
             }
 
-            pub fn permit_to_send_out(mut self) -> Self {
+            pub fn capture_msg_and_let_it_flight_on(mut self) -> Self {
```
Comment (dnwiebe) [inline node/src/test_utils/mod.rs]: Probably should be "fly"

File: node/src/accountant/mod.rs [hunk 7]
```diff
@@ -645,18 +645,22 @@ impl Accountant {
         })
     }
 
-    fn handle_payable_payment_setup(&mut self, msg: PayablePaymentSetup) {
-        let bb_instructions = match self.scanners.payable.try_softly(msg, &self.logger) {
+    fn handle_payable_payment_setup(&mut self, msg: PayablePaymentsSetup) {
```
Comment (dnwiebe) [inline node/src/accountant/mod.rs:648]: `handle_payable_payments_setup` plural

File: node/src/accountant/mod.rs [hunk 8]
```diff
@@ -645,18 +645,22 @@ impl Accountant {
...
+        let bb_instructions = match self
```
Comment (dnwiebe) [inline node/src/accountant/mod.rs]: Would be clearer as `blockchain_bridge_instructions`

File: node/src/accountant/mod.rs [hunk 9]
```diff
@@ -1516,21 +1550,23 @@ mod tests {
         assert_eq!(system.run(), 0);
         let after = SystemTime::now();
         let mut adjust_payments_params = adjust_payments_params_arc.lock().unwrap();
-        let (cwbqp_msg, captured_now, logger_clone) = adjust_payments_params.remove(0);
+        let (actual_awaited_adjustment, captured_now, logger_clone) =
+            adjust_payments_params.remove(0);
+        assert!(adjust_payments_params.is_empty());
```
Comment (dnwiebe) [inline node/src/accountant/mod.rs]: Moving this back down to the end of this batch of assertions will give better failure messages.

**Pull request closed without merging**